core/domain/suggestion_services.py

```diff
@@ -308,6 +310,198 @@ def get_suggestion_by_id(
     return get_suggestion_from_model(model) if model else None
 
 
+@overload
+def get_translation_contribution_stats_models(
+    stats_ids: List[str], *, strict: Literal[True]
+) -> List[suggestion_models.TranslationContributionStatsModel]: ...
+
+
+@overload
+def get_translation_contribution_stats_models(
+    stats_ids: List[str]
+) -> List[Optional[suggestion_models.TranslationContributionStatsModel]]: ...
```

Comment:
```suggestion
@overload
def get_translation_contribution_stats_models(
    stats_ids: List[str]
) -> List[suggestion_models.TranslationContributionStatsModel]: ...
```

because the default value for `strict` is `True`.

Reply:
Done